en/console-and-shells/completion-shell.rst

```diff
@@ -94,20 +106,50 @@ The following bash example comes from the original author::
             opts=$(${cake} Completion subcommands $prev)
             COMPREPLY=( $(compgen -W "${opts}" -- ${cur}) )
             if [[ $COMPREPLY = "" ]] ; then
-                COMPREPLY=( $(compgen -df -- ${cur}) )
+                _filedir
```

Comment:
The reason I changed this is that the _filedir function deals perfectly with filesystem browsing : it adds a slash after an autocomplete on a dir, suggest the good dir as if you were doing it with ``cd`` and suggest files and so on.
The current snippet suggests the dirs correctly but adds a space after the completion.